packages/components/src/ListView/Items/Item/Item.component.js

```diff
@@ -66,6 +68,13 @@ class Item extends Component {
 					label={searchCriteria ? getSearchedLabel(item.label) : item.label}
 					aria-label={ariaLabel}
 				/>
+				{item.icon && (
+					<TooltipTrigger label={item.icon.title} tooltipPlacement="bottom">
+						<span>
+							<Icon {...item.icon} aria-hidden="false" />
```

Comment:
Icons are aria-hidden by default

Reply:
Are the icons only decorative ? For example next to the displayed text.
If so, it should stay hidden you can remove this extra aria-hidden (default in Icon is true as Fabien said).
If not, they must have a text-alternative.

Reply:
yes icon is decorative with a hint